pyro/primitives.py

```diff
@@ -0,0 +1,439 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
this file was moved verbatim from `pyro/__init__.py` right?

Reply:
That's correct, this should be a straight migration.